packages/query/src/calendar/index.ts

```
@@ -0,0 +1,43 @@
+import { useQuery } from "@tanstack/react-query"
+import NepaliDate from "nepali-datetime"
```

Biome

[error] All these imports are only used as types.

Importing the types with import type ensures that they are removed by the compilers and avoids loading unnecessary modules.
Safe fix: Use import type.


(lint/style/useImportType)